data-raw/ald_demo.csv

```diff
@@ -68,19 +68,19 @@ affinity renewables inc.,power,renewablescap,MW,2037,20874.066126101905,NA,AS,MW
 affinity renewables inc.,power,renewablescap,MW,2038,21270.991879608748,NA,AS,MW,6,TRUE,FALSE,1994Q1
 affinity renewables inc.,power,renewablescap,MW,2039,21667.917633115587,NA,AS,MW,6,TRUE,FALSE,1994Q1
 affinity renewables inc.,power,renewablescap,MW,2040,22064.843386622426,NA,AS,MW,6,TRUE,FALSE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2013,32422050.652607474,4765807.852467671,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2014,31919907.220158126,4406059.615706608,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2015,31417763.787708778,4046311.378945545,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2016,30915620.355259433,3686563.1421844815,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2017,30413476.922810085,3326814.9054234186,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2018,29911333.490360737,2967066.6686623553,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2019,29409190.05791139,2607318.431901292,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2020,28907046.62546204,2247570.1951402286,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2021,28404903.193012692,1887821.9583791657,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2022,27902759.760563344,1528073.7216181024,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2023,27400616.328113995,1168325.484857039,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2024,26898472.89566465,808577.2480959762,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2025,26396329.463215303,448829.01133491244,CD,CM,1,TRUE,TRUE,1994Q1
+africa oil corp,oil and gas,oil,GJ per day,2013,32422050.652607474,4765807.852467671,CD,CM,1,TRUE,TRUE,1994Q1
```

Comment:
Thanks!
We'll also need to update data/ald_demo by sourcing data-raw/ald_demo.R.
That should make the regression test to fail. So we'll also need to temporarily set `update = TRUE`, then run tests twice and set `update = FALSE` back again.

Reply:
i knew i forgot something! havent looked at dataraw in a while haha